packages/labs/testing/src/lib/fixtures/csrFixture.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {render} from 'lit';
+import type {LitElement, TemplateResult} from 'lit';
+import type {FixtureOption} from './fixtureOption.js';
+
+/**
+ * Renders the provided lit-html template with a Lit element client-side.
+ *
+ * @param {TemplateResult} template - lit-html template. Must contain a single
+ * top level custom element.
+ * @param {string[]} option.modules - Path to custom element definition modules
```

Comment:
In VSCode this annotation style doesn't quite seem to work -- only the `Path to custom element definition modules` part renders when you are typing the parameter. I think if we annotate the `FixtureOption` properties directly on that type signature, it will work better.